### DISCUSSION TITLE
[Web] Cannot type anything on web

### DISCUSSION THREAD

**Discussion by amitozalvo** [singerdmx/flutter-quill#615] [Web]
The editor is broken on web.
even when it's focused, nothing is registered..
I'm on stable version of flutter and on the updated version of flutter-quill

Ive seen another issue like this which is closed for no reason...

**Comment by NicolasDionB** [comment]
Maybe this discussion can help? The workaround worked for me. #452 

Also, did you test on different browsers/platform?

**Reply by NicolasDionB** [reply]
@amitozalvo Actually, I committed the workaround in the official release. Update to the latest 3.8.2 to see if it fixes your issue.
